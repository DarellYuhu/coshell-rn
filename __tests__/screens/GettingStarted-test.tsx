import GettingStarted from "@/app/(initial)";
import { render } from "@testing-library/react-native";

describe("Test getting started screen", () => {
  it("should render correctly", () => {
    const { getByText } = render(<GettingStarted />);

    getByText("GET STARTED");
  });

  it("should render correctly with snapshot", () => {
    const tree = render(<GettingStarted />).toJSON;

    expect(tree).toMatchSnapshot();
  });
});
