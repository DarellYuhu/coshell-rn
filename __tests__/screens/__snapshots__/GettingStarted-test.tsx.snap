// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test getting started screen should render correctly with snapshot 1`] = `[Function]`;
